src/nvim/ex_getln.c

```diff
@@ -2517,6 +2517,7 @@ static void cmdpreview_show(CommandLineState *s)
   // If preview callback returned 0, update screen to clear remnants of an earlier preview.
   if (cmdpreview_type == 0) {
     update_screen(SOME_VALID);
+    cmdpreview = false;
```

Comment:
Minor nitpick: Let's put this line before `update_screen` to keep consistency with other places where cmdpreview is set to false.

Reply:
Indeed this should be better.